snippets/base/migrations/0027_auto_20180425_1041.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.12 on 2018-04-25 10:41
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from django_mysql.models.functions import AsType, ColumnAdd
+
+
+def set_is_developer(apps, schema_editor):
+    """Migrate existing snippets that used about:accounts link filtering."""
+    Snippet = apps.get_model('base', 'Snippet')
+    Snippet.objects.update(
+        client_options=ColumnAdd(
```

Comment:
Never used Dynamic Columns before. Very cool.

Reply:
Yeah, they are great for this kind of things